docs/_sidebar.md

```diff
@@ -25,21 +25,22 @@
 
   </details>
 
-  <details open>
```

Comment:
why the open is removed? I thought we wanted to give emphasis to the Neo 3 Guides.

Reply:
just a personal preference. i'll remove the change then.

Reply:
@mialbu and I discussed this in the past, and we would prefer to leave it open.